nilearn/_utils/data_gen.py

```diff
@@ -751,6 +750,25 @@ def basic_confounds(length, random_state=0):
     return confounds
 
 
+def add_metadata_to_bids_dataset(bids_path,
+                                     metadata,
+                                     json_file=None):
+    if json_file is None:
+        json_file = (Path(bids_path) / 
+                        'derivatives' / 
+                        'sub-01' / 
+                        'ses-01' / 
```

Comment:
On which grounding are this subject and session id chosen ?

Reply:
I guess it's because this is only used for testing. Maybe indicate this in a short docstring.

Reply:
good point